lib/ansible/config/base.yml

```diff
@@ -1102,10 +1102,11 @@ DEFAULT_TIMEOUT:
   - {key: timeout, section: defaults}
   type: integer
 DEFAULT_TRANSPORT:
-  # note that ssh_utils refs this and needs to be updated if removed
   name: Connection plugin
-  default: smart
-  description: "Default connection plugin to use, the 'smart' option will toggle between 'ssh' and 'paramiko' depending on controller OS and ssh versions"
+  default: ssh
+  description:
+    - Can be any connection plugin available to your ansible installation.
+    - There is also a (DEPRECATED) special 'smart' option, that will toggle between 'ssh' and 'paramiko' depending on controller OS and ssh versions.
```

Comment:
If it's being deprecated should we even call it out here?

Reply:
it was documented here before, we should still let them know it is valid. Imagine someone walks into existing setup with this set in configuration to 'smart' .. they would not see how that translates into a plugin.

Reply:
My only concern is this deprecation notice stays beyond the point when it is removed. We have no way to automate the checks to ensure it is also removed at the same time. If we keep it here I would at least specify the version in which it will be removed.